packages/v4/src/content/design-guidelines/components/tree-view/tree-view.md

```diff
@@ -3,5 +3,92 @@ id: Tree view
 section: components
 ---
 
-We’re working on some guidelines for this component.
-If you have any questions in the meantime, feel free to [get in touch with us.](/get-in-touch)
+A **tree view** is a structure that displays data in a hierarchical view. It can be used in a [dropdown](/components/dropdown), [drawer](/components/drawer), [primary-detail](/demos/primary-detail), [modal](/components/modal), or [wizard](/components/wizard).
+
+A default tree view provides a tree structure with items and an arrow to expand or collapse child items. Every tree view can be extended with these [optional elements](#tree-view-capabilities).
+
+## Tree view elements
+
+A default tree view consists of 5 elements:
+
+<img src="./img/tree-view-elements.png" width="332" />
+
+1. **Expand/collapse button:** A button that allows you to expand or collapse child nodes 
+2. **Parent node:** A node that contains other items (for example, a folder that contains multiple documents)
+3. **Child node:** A node within the parent node (for example, a document within a folder)
+4. **Leaf node:** The last item in a tree, which doesn't have any child nodes
+5. **Truncation indicator:** A control that automatically shortens long names 
```

Comment:
I wouldn't call this a "control" I would just say "Longer names that do not fit in the node width will be indicated with ellipses" or something like that

Reply:
To keep it consistent with how the other options are written, we could try, "Truncation indicator: Ellipses indicating truncated text for longer names that don't fit in the node width"